server/models/newsletter_subscriber.py

```
@@ -4,19 +4,32 @@
 
-class NewsletterSubscribers(db.Model):
+class NewsletterSubscriber(db.Model):
     id = db.Column(db.Integer, primary_key=True)
-    email = db.Column(db.String(120), unique=True, nullable=False)
+    email = db.Column(db.String(254), unique=True, nullable=False)
     subscription_date = db.Column(
-        db.DateTime(timezone=True),
-        default=lambda: datetime.now(timezone.utc),
-        nullable=False,
-    )
+    db.DateTime(timezone=True),
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)